src/AutoRest.CSharp/Common/Output/Models/Shared/KnownParameters.cs

```diff
@@ -31,7 +31,7 @@ internal static class KnownParameters
 
         public static readonly Parameter RequestContext = new("context", "The request context, which can override default behaviors on the request on a per-call basis.", RequestContextNullableType, Constant.Default(RequestContextNullableType), false);
 
-        public static readonly Parameter WaitForCompletion = new("waitForCompletion", "true if the method should wait to return until the long-running operation has completed on the service; false if it should return after starting the operation. For more information on long-running operations, please see <see href=\"https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/LongRunningOperations.md\"> Azure.Core Long-Running Operation samples</see>.", new CSharpType(typeof(bool)), null, false);
+        public static readonly Parameter WaitForCompletion = new("waitUntil", "Completed if the method should wait to return until the long-running operation has completed on the service; Started if it should return after starting the operation. For more information on long-running operations, please see <see href=\"https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/LongRunningOperations.md\"> Azure.Core Long-Running Operation samples</see>.", new CSharpType(typeof(WaitUntil)), null, false);
```

Comment:
Is it possible that we could add some code links here to `WaitUntil.Completed` and `WaitUntil.Started`?
Like:
```suggestion
        public static readonly Parameter WaitForCompletion = new("waitUntil", "\"F:WaitUntil.Completed\" if the method should wait to return until the long-running operation has completed on the service; \"F:WaitUntil.Started\" if it should return after starting the operation. For more information on long-running operations, please see <see href=\"https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/LongRunningOperations.md\"> Azure.Core Long-Running Operation samples</see>.", new CSharpType(typeof(WaitUntil)), null, false);
```
This format comes from this document: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/